pxr/base/tf/testenv/pathUtils.cpp

```diff
@@ -104,11 +104,11 @@ TestTfRealPath()
 
     if (testSymlinks) {
         // Leaf dir is symlink
-        TF_AXIOM(TfRealPath("d", true) == TfAbsPath("subdir"));
+        TF_AXIOM(TfNormPath(TfRealPath("d", true)) == TfNormPath(TfAbsPath("subdir")));
```

Comment:
Thanks a lot for your comments. We'll wrap these changes under ARCH_OS_WINDOWS and add some comments as well.

The reason is that TfRealPath on windows explicitly lower-cases drive letters if the given path contains symlinks, otherwise it returns the same as os.path.abspath. The similar workaround is already used in [pxr/base/tf/testenv/testTfPathUtils.py](https://github.com/PixarAnimationStudios/OpenUSD/blob/release/pxr/base/tf/testenv/testTfPathUtils.py#L58-L68)

Reply:
Branch updated. @tallytalwar - let us know if this works or if you need any more follow-up.